### PR TITLE
Issue 2168 - cleanup appendix

### PR DESCRIPTION
**Pull request by Stackblocks**
- Moved H1 over H2 in the appendix-shapes
- Rename old appendix-specific components to be generic
- Swapped in H1Component
- Replaced the chemistryTitleColor used with a generic appendix version
- First step towards breaking border settings out of titles
and into their own components.

### CODE REVIEW

File: styles/output/chemistry-pdf.css [hunk 1]
```diff
@@ -882,6 +919,7 @@ a {
   font-weight: bold;
   text-transform: uppercase; }
 
+>>>>>>> ignoble.icarus
```
Comment (ajluk) [inline styles/output/chemistry-pdf.css:922]: @Stackblocks There are some unresolved conflicts.

File: styles/output/chemistry-pdf.css.map [hunk 2]
```diff
@@ -116,5 +116,9 @@
...
+>>>>>>> ignoble.icarus
```
Comment (ajluk) [inline styles/output/chemistry-pdf.css.map:123]: See my comment above.

File: styles/templates/template1/_colors.scss [hunk 3]
```diff
@@ -99,5 +105,6 @@
     visualColor: (_ref: "scheme:::B2"),
     watchPhysicsColor: (_ref: "scheme:::B2"),
     workInPhysicsColor: (_ref: "scheme:::BR1"),
+>>>>>>> ignoble.icarus
```
Comment (ajluk) [inline styles/templates/template1/_colors.scss:108]: Same here.

File: styles/output/chemistry-pdf.css (hunk 1 above)
Comment (Stackblocks) [reply styles/output/chemistry-pdf.css:922]: Thanks for catching these! I guess this is what happens when I try to resolve the conflicts by hand for the first time, lol. I'll resolve these and re-commit. ✅

File: styles/designs/carnival/parts/_titles-components.scss [hunk 4]
```diff
@@ -9,57 +9,57 @@ $H1Component:(
...
+        border-bottom-color: enum('ValueSet:::OPTIONAL'),
+        border-bottom-width: enum('ValueSet:::OPTIONAL'),
+        border-bottom-style: enum('ValueSet:::OPTIONAL'),
         margin-bottom: v-spacing(2),
     )
 );
 
 
 //h1 numbered (appendix)
 //todo rename this to not be appendix specific
```
Comment (brittany-johnson) [inline styles/designs/carnival/parts/_titles-components.scss:26]: I think we're safe to remove the todo comment

**Pull request merged**